ovos_utils/system.py

```diff
@@ -177,51 +183,25 @@ def check_service_active(service_name, sudo=False, user=False) -> bool:
     state = subprocess.run(status_command, shell=True).returncode
     return state == 0
 
-
-# platform fingerprinting
-def set_root_path(path):
-    global _USER_DEFINED_ROOT
-    _USER_DEFINED_ROOT = path
-    LOG.info(f"mycroft root set to {path}")
-
-
-def find_root_from_sys_path():
-    """Find mycroft root folder from sys.path, eg. venv site-packages."""
-    for p in [path for path in sys.path if path != '']:
-        if exists(join(p, 'mycroft', 'configuration', 'mycroft.conf')):
-            return p
-    else:
-        return None
-
-
-def find_root_from_sitepackages():
-    """Find root from system or venv's sitepackages."""
-    site = sysconfig.get_paths()['platlib']
-    if exists(join(site, 'mycroft', 'configuration', 'mycroft.conf')):
-        return site
-    else:
-        return None
-
-
-def search_mycroft_core_location():
-    """Check python path (.venv), system packages and finally known mycroft
-    locations."""
-    # downstream wants to override the root location
-    if _USER_DEFINED_ROOT:
-        return _USER_DEFINED_ROOT
-    # if we are in a .venv that should take precedence over everything else
-    if find_root_from_sitepackages():
-        return find_root_from_sitepackages()
-    # if there is a system wide install that should take precedence over
-    # hardcoded locations
-    elif find_root_from_sys_path():
-        return find_root_from_sys_path()
-    # finally look at default locations
-    for p in MycroftRootLocations:
-        if os.path.isdir(p):
-            return p
-    return None
```

Comment:
These weren't previously marked for deprecation. Should the references stay and return `None` with a deprecation notice to prevent import errors?

Reply:
these dont really apply anymore and the results arent meaningful, nothing in core uses it either, these are old utils from when we used mycroft-core (pre ovos-core)

dont think any of those returns a valid value at all anymore or has a sane replacement for current architecture, once we split into several services this stopped making sense

i think these were really meant to be user facing either, just for internal lib usage?